pom.xml

```diff
@@ -26,9 +26,36 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.uber.tchannel</groupId>
-    <artifactId>tchannel</artifactId>
+    <artifactId>tchannel-java</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
SNAPSHOT usually is version before 1.0. But that's just convention. If java tchannel isn't ready or stable yet, we can still have SNAPSHOT.